ci-operator/config/openshift/cluster-api/openshift-cluster-api-release-1.0.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.16:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+binary_build_commands: make manager-core
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- build_args:
+  - name: builder_image
+    value: docker.io/library/golang:1.16.8
```

Comment:
Should this be `registry.ci.openshift.org/openshift/release:golang-1.16`?

Reply:
I set the current one because otherwise the built failed with some go vendor issue, I assume because the config is different. Not sure it makes a big difference since then the dockerfile use another image to contain the final binary https://github.com/openshift/cluster-api/blob/cluster-api-0.1.0/Dockerfile#L16-L28

I was planning to get this in as first step then follow up to swap to the right images and maybe change the dockerfile but already driven CI

Reply:
This will end up flaking because docker.io pulls rate ratelimited. IIRC the image dan linked sets the `GOFLAGS=-mod=vendor` env var or sth like that which could explain the issues you are seeing

Reply:
Right https://github.com/openshift/release/blob/c32756180704c239f76063c8d39bc464ea7d2374[…]ters/01_cluster/ci/_origin-release-build/golang-1.16/Dockerfile

I updated to use the right image an line the Docker file to set the Env variable.